x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/results/FeatureImportance.java

```diff
@@ -104,22 +109,28 @@ public String getFeatureName() {
 
     @Override
     public void writeTo(StreamOutput out) throws IOException {
-        out.writeString(this.featureName);
-        out.writeDouble(this.importance);
-        out.writeBoolean(this.classImportance != null);
-        if (this.classImportance != null) {
+        out.writeString(featureName);
+        if (out.getVersion().onOrAfter(Version.V_8_0_0)) {
+            out.writeOptionalDouble(importance);
+        } else {
+            out.writeDouble(importance);
```

Comment:
couldn't this blow up? we need to somehow require that `importance` here is not null.

I think the way to do that is to run 
```
            classImportance.stream().mapToDouble(ClassImportance::getImportance).map(Math::abs).sum()
```
if it is null. If classImportance is null && importance is null, then maybe just return a 0.0

Reply:
Indeed!